cameraview/src/main/java/com/otaliastudios/cameraview/engine/Camera1Engine.java

```diff
@@ -745,6 +752,24 @@ private boolean applyPreviewFrameRate(@NonNull Camera.Parameters params,
         return false;
     }
 
+    private void sortRanges(List<int[]> fpsRanges) {
+        if (mPreviewFrameRateExact) { // sort by range width in ascending order
```

Comment:
Looks great! Can we add the case where the desired frame rate is 0 (unspecified)? In this case, I would like to take **wide** intervals first. Something like...

```java
private void sortRanges(List<int[]> fpsRanges, float desiredFps) {
    if (mPreviewFrameRateExact && desiredFps != 0F) { 
        // sort ascending...
    } else {
       // sort descending...
    }
}
```